src/sql/azdata.proposed.d.ts

```diff
@@ -1333,6 +1337,10 @@ declare module 'azdata' {
 			 * Cost string for the node
 			 */
 			costDisplayString: string;
+			/**
+			 * Row metrics for the node
+			 */
+			rowMetrics: Dictionary;
```

Comment:
please add an example in the comments

Reply:
comment not resolved.

Reply:
Removed `Dictionary` and replaced with `CostMetrics` interface